pybsn/__init__.py

```diff
@@ -205,6 +373,12 @@ def guess_url(session, host, validate_path="/api/v1/auth/healthy"):
     raise Exception("Could not find available BigDB service on {}".format(host))
 
 def attempt_login(session, url, username, password):
```

Comment:
nit: wonder if this should be prefixed with `_`? I don't expect many folks to call this and not call `connect`